apps/server/src/shared/repo/teams/teams.repo.ts

```diff
@@ -25,6 +26,17 @@ export class TeamsRepo extends BaseRepo<Team> {
 		return team;
 	}
 
+	/**
+	 * Finds partly teams which the user is a member.
```

Comment:
I dont understand the sentence. are we only finding a part of the teams? or parts of each team?

Reply:
I didnt adjusted the doc. Done